src/xccl/ProcessGroupXCCL.hpp

```diff
@@ -108,7 +117,22 @@ class TORCH_API ProcessGroupXCCL : public Backend {
     friend class ProcessGroupXCCL;
   };
 
-  ProcessGroupXCCL(const c10::intrusive_ptr<Store>& store, int rank, int size);
+  struct Options : public Backend::Options {
+    explicit Options();
+
+    static c10::intrusive_ptr<Options> create() {
+      return c10::make_intrusive<Options>();
+    }
+
+    std::vector<uint64_t> global_ranks_in_group;
+    std::string group_name;
```

Comment:
I see you also bind those to python in public stock PR. I'm just wondering if it's more reasonable to move definitions to `Backend::Options` instead of keeping separately in each backend like NCCL/XCCL.